test-webhook.js

```
@@ -0,0 +1,118 @@
+// Simple test script to verify webhook endpoint functionality
+// Run with: node test-webhook.js
+
+const crypto = require('crypto');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)